### ISSUE TITLE
Continue investigating triage-party for use within the release team

### ISSUE THREAD

**Issue by markyjackson-taulia** [kubernetes/sig-release#1073]
#### What would you like to be added:
Continue investigating triage-party for use within the release team
#### Why is this needed:
This tool will help with timely triaging as well and the bug-triage team during the release cycle.
@markyjackson-taulia did a presentation with @tstromberg during the release SIG meeting on 4/28/2020

**Comment by markyjackson-taulia** [comment]
priority/important-soon

**Comment by markyjackson-taulia** [comment]
@Verolop @xmudrii We can track our work with this issue and breakout where needed

**Comment by markyjackson-taulia** [comment]
Task breakout meeting set for 5/15. Working doc is here: https://docs.google.com/document/d/1p9XYRMT2dmIYBZxM9OwINjeRAGo5BwUeqZ8VTkklb0A/edit?usp=sharing

**Comment by markyjackson-taulia** [comment]
My apologies for not updating this issue in a timely manner. 
@Verolop @xmudrii and I met on Monday to cover the tasks at hand. The doc here has been updated to reflect those items: https://docs.google.com/document/d/1p9XYRMT2dmIYBZxM9OwINjeRAGo5BwUeqZ8VTkklb0A/edit?usp=sharing

**Comment by justaugustus** [comment]
Keep up the great work!

**Comment by ameukam** [comment]
@Verolop @xmudrii FYI: https://github.com/kubernetes/k8s.io/issues/906

**Comment by markyjackson-taulia** [comment]
This can be closed as @ameukam is working through the final bit with dns but this investigation is completed.
/close

**Comment by justaugustus** [comment]
Let's continue to track this until Triage Party is live for SIG Release.
/reopen

**Comment by markyjackson-taulia** [comment]
My bad, should have talked to more people before just closing. Won't happen again.

**Comment by justaugustus** [comment]
Not a big deal. Just want to make sure we close the loop. :)
/assign @ameukam @Verolop @xmudrii

**Comment by tstromberg** [comment]
Anything I can do to help support Triage Party in Kubernetes?

We just released Triage Party v1.3.0, which is mostly a stability release.

**Comment by LappleApple** [comment]
Heya, my effort to drive a layout prototype for the RT's version got pushed back due to a few personal and K8s-related developments, but I'll be on this topic again over the next three days.

**Comment by LappleApple** [comment]
Update: I've shared [the prototype doc](https://docs.google.com/document/d/1Yen26GS7VBX5CydwT8lk2j1m6yyllZaqd-oXjyaEGYE/edit#heading=h.e87k3ltggrs8) in my regular Monday session with the Release leads and then yesterday with a broader group at the 1.19 Release Team retro. Seems like we could move forward with setting up the details of the board.

**Comment by LappleApple** [comment]
@ameukam @Verolop Hey, anything more to do here? :)

**Comment by ameukam** [comment]
We now have an instance (https://release.triage.k8s.io) running on `aaa` cluster : https://github.com/kubernetes/k8s.io/tree/main/triage-party/release-team. Over time, we are facing some issues but `tstromberg` is working to address them (Thank you! ❤️).

IIRC, @Verolop is working on a configuration for TP. Maybe we should adjust it to `v1.22`. 
IMHO, next step would be collect wishes from SIG-Release about what need to be add. (can be a separate issue)

**Comment by LappleApple** [comment]
Hey folks, given the progress made last week with configuration can we close this item now? cc @cpanato @kubernetes/sig-release-leads

**Comment by cpanato** [comment]
@LappleApple I think we can close this for now and open new ones if needed after we play a bit with it

Did a small or to adjust some config and after that, we can close this

**Comment by ameukam** [comment]
+1 for closing once https://github.com/kubernetes/k8s.io/pull/1852 is merged and deployed.

**Comment by ameukam** [comment]
Oups. Bot took over when I mentioned the PR. Sorry.
